seml/config.py

```diff
@@ -384,13 +384,15 @@ def read_config(config_path):
 
     determine_executable_and_working_dir(config_path, seml_dict)
 
-    if 'slurm' in config_dict:
+    if ('slurm' in config_dict) and (config_dict['slurm'] is not None):
```

Comment:
very small nit: we do not use parentheses if they are not necessary. I'd remove these for consistency.